Sources/MessagingInApp/Type/InAppMessage.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import Gist
+
+typealias GistMessage = Message
+
+public struct InAppMessage: Equatable {
+    public let instanceId: String
+    public let messageId: String
+    public let deliveryId: String // (Currently taken from Gist's campaignId property)
+
+    internal init(gistMessage: GistMessage) {
+        self.instanceId = gistMessage.instanceId
+        self.messageId = gistMessage.messageId
+        // The Gist SDK source code always populates the campaign-id. Having a nil campaignId is only common using the
+        // `public init()` for Message.
```

Comment:
Though very low probability of this happening but in case it happens then force unwrapping will make the app crash. I am trying to think of a use case but all I can think right now is if customer create own message instance but not sure if that would ever happen !

Reply:
I was not aware of this, but it does look like sending test in-app messages can make `deliveryId` null. https://github.com/customerio/customerio-ios/pull/209/files#r1020570037

Reply:
I made `deliveryId` optional.